README.md

```diff
@@ -62,7 +62,7 @@ Our implementation provides an easy to use interface to build custom optimizatio
 - [Second-order nonlinear optimizers](https://github.com/facebookresearch/theseus/tree/main/theseus/optimizer/nonlinear)
     - Gauss-Newton, Levenberg–Marquardt
 - [Linear solvers](https://github.com/facebookresearch/theseus/tree/main/theseus/optimizer/linear)
-    - Dense: Cholesky, LU; Sparse: CHOLMOD, LU
+    - Dense: Cholesky, LU; Sparse: CHOLMOD, LU, [BaSpaCho](https://github.com/facebookresearch/baspacho)
```

Comment:
Should we add next to Sparsse LU (GPU only), so it's clear all others support both CPU and GPU?